introduction.md

```diff
@@ -0,0 +1,7 @@
+Hello, I'm a person living at a place trying to master GitHub. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.